github/repos_collaborators.go

```diff
@@ -10,10 +10,25 @@ import (
 	"fmt"
 )
 
+// ListCollaboratorsOptions specifies the optional parameters to the
+// RepositoriesService.ListCollaborators method.
+type ListCollaboratorsOptions struct {
+	ListOptions
```

Comment:
Let me know if you find evidence to the contrary, but look at other similar Options structs, I think we tend to put `ListOptions` at the very bottom. E.g.:

https://github.com/google/go-github/blob/78072f320213a7e011b5fb2700bcb9eb90efec37/github/repos.go#L123-L156

Let's do that here too for consistency.